### ISSUE TITLE
Ensure correct hierarchical order of headings in Clarity wizard component

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#3994]
## Summary
Title of Clarity wizard is heading with tag &lt;h3&gt;, so is every page title.
To ensure the hierarchical order of headings h1 and h2 tags should be used.

* What is the change?
Use h1 tag for the wizard title and h2 for the page title
* Why should it go in Clarity?
Accessibility enhancement

* Does this change impact existing behaviors? If so how?
No
* If this change introduces a new behavior, is this behavior accessible?
Yes

## Use case

N/A

## Examples

https://v2.clarity.design/wizards

## Workarounds

N/A

**Comment by mathisscott** [comment]
Marking this as a breaking change because it constitutes a breaking change to the HTML DOM structure that would make it incompatible with previous versions of `@clr/ui` CSS.
